lldb/source/Plugins/Process/elf-core/ProcessElfCore.h

```diff
@@ -158,6 +165,16 @@ class ProcessElfCore : public lldb_private::PostMortemProcess {
   // Returns number of thread contexts stored in the core file
   uint32_t GetNumThreadContexts();
 
+  // Populate gnu uuid for each NT_FILE entry
+  void UpdateBuildIdForNTFileEntries();
+
+  // Returns the value of certain type of note of a given start address
+  std::optional<lldb_private::UUID> FindNote(const lldb::addr_t address,
```

Comment:
`const` on a value parameter (pointer and reference parameters are of course different) declaration don't actually do anything, and I don't believe we usually use them.

If you really want to you can still make them `const` in [the function definition](https://godbolt.org/z/PxM739P84) even if they're not const in the header (though we usually don't do that either).